node/src/chain_specs/manta.rs

```diff
@@ -198,4 +222,203 @@ pub fn manta_testnet_config() -> Result<MantaChainSpec, String> {
 /// Returns the Manta mainnet chainspec.
 pub fn manta_config() -> Result<MantaChainSpec, String> {
     MantaChainSpec::from_json_bytes(&include_bytes!("../../../genesis/manta-genesis.json")[..])
+    // Ok(mainnet::manta_mainnet_config())
```

Comment:
Remove commented line?

Reply:
this is a bit of a topic for discussion, i want to preserve the code that generated the genesis json.
And tbh I don't even know the reason why we do this file read in the first place.

@ghzlatarev do you know the history?

Reply:
When we make it into a file we can use it with any version of the node and it should start from the original runtime.
If it's in-place it would start with the latest runtime version.
I think that's the reason.
Also maybe obfuscating some account ids.

Reply:
makes sense, the RT blob gets built into the file.

This actually means we can only generate the config after bumping spec versions, otherwise it'll report as `manta/4000`

I'm wondering how to preserve this code tho, maybe splitting the module into its own file?

Reply:
I think its because the genesis file must be etched into stone. 

Aka if we change the genesis than the node can't sync to manta network as it has a different genesis block

Reply:
the line will be needed in #957 and we can use that issue to figure out where to put the code that generated it.